web/routing.go

```diff
@@ -115,7 +115,7 @@ func newRenderer(assetsPath string) (*renderer, error) {
 func SetupAppsHandler(appsHandler echo.HandlerFunc) echo.HandlerFunc {
 	secure := middlewares.Secure(&middlewares.SecureConfig{
 		HSTSMaxAge:    hstsMaxAge,
-		CSPDefaultSrc: []middlewares.CSPSource{middlewares.CSPSrcSelf, middlewares.CSPSrcParent},
+		CSPDefaultSrc: []middlewares.CSPSource{middlewares.CSPSrcSelf, middlewares.CSPSrcParent, middlewares.CSPSrcWhitelist},
```

Comment:
I not totally certain, but they are not images at least, both are json APIs :

nominatim.openstreetmap.org :
```
curl 'http://nominatim.openstreetmap.org/search?format=json&q=2%20BOULEVARD%20DES%20BELGES+LYON+FRANCE' -H 'Host: nominatim.openstreetmap.org' -H 'User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:54.0) Gecko/20100101 Firefox/54.0' -H 'Accept: */*' -H 'Accept-Language: fr,fr-FR;q=0.8,en-US;q=0.5,en;q=0.3' --compressed -H 'Referer: http://app.cozy.tools:8080/' -H 'Origin: http://app.cozy.tools:8080' -H 'Connection: keep-alive' -H 'Cache-Control: max-age=0'
```

osmbuildings.org : 
```
curl 'http://c.data.osmbuildings.org/0.2/anonymous/tile/15/16824/11686.json' -H 'Host: c.data.osmbuildings.org' -H 'User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:54.0) Gecko/20100101 Firefox/54.0' -H 'Accept: */*' -H 'Accept-Language: fr,fr-FR;q=0.8,en-US;q=0.5,en;q=0.3' --compressed -H 'Referer: http://app.cozy.tools:8080/' -H 'Origin: http://app.cozy.tools:8080' -H 'Connection: keep-alive'
```